### PR TITLE
1.4.7, 1.5.1, istio-security-2020-004

### PR DESCRIPTION
**Pull request by fpesce**

### CODE REVIEW

**Comment by fpesce** [comment]
This PR replaces #6963 and #6952

File: content/en/news/releases/1.5.x/announcing-1.5.1/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+This release contains bug fixes to improve robustness and fixes for the security vulnerabilities described in [our March 25th, 2020 news post](/news/security/istio-security-2020-004). This release note describes what’s different between Istio 1.5.0 and Istio 1.5.1.
+
+{{< relnote >}}
+
+## Security update
+
+- **ISTIO-SECURITY-2020-004** Istio uses a hard coded `signing_key` for Kiali.
+
+__[CVE-2020-1764](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-1764)__: Istio uses a default `signing key` to install Kiali. This can allow an attacker with access to Kiali to bypass authentication and gain administrative privileges over Istio.
+In addition, another CVE is fixed in this release, described in the Kiali 1.15.1 [release](https://github.com/kiali/kiali/releases/tag/v1.15.1).
```
Comment (brian-avery) [inline content/en/news/releases/1.5.x/announcing-1.5.1/index.md]: This link is wrong. Should be https://kiali.io/news/security-bulletins/kiali-security-001/

File: content/en/news/releases/1.4.x/announcing-1.4.7/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+This release contains fixes for the security vulnerabilities described in [our March 25th, 2020 news post](/news/security/istio-security-2020-004). This release note describes what’s different between Istio 1.4.6 and Istio 1.4.7.
+
+{{< relnote >}}
+
+## Security Update
+
+- **ISTIO-SECURITY-2020-004** Istio uses a hard coded `signing_key` for Kiali.
+
+__[CVE-2020-1764](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-1764)__: Istio uses a default `signing key` to install Kiali. This can allow an attacker with access to Kiali to bypass authentication and gain administrative privileges over Istio.
+In addition, another CVE is fixed in this release, described in the Kiali 1.15.1 [release](https://github.com/kiali/kiali/releases/tag/v1.15.1).
```
Comment (fpesce) [inline content/en/news/releases/1.4.x/announcing-1.4.7/index.md]:
```suggestion
In addition, another CVE is fixed in this release, described in the Kiali 1.15.1 [release](https://kiali.io/news/security-bulletins/kiali-security-001/).
```

File: content/en/news/releases/1.5.x/announcing-1.5.1/index.md (hunk 1 above)
Comment (fpesce) [inline content/en/news/releases/1.5.x/announcing-1.5.1/index.md]:
```suggestion
In addition, another CVE is fixed in this release, described in the Kiali 1.15.1 [release](https://kiali.io/news/security-bulletins/kiali-security-001/).
```

File: content/en/news/security/istio-security-2020-004/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+To determine if your signing key is unset, run this command and check for blank output:
+
+{{< text bash >}}
+$ kubectl get cm kiali -n istio-system -o yaml | grep signing_key
+{{< /text >}}
+
+## Mitigation
+
+* For Istio 1.4.x deployments: update to [Istio 1.4.7](/news/releases/1.4.x/announcing-1.4.7) or later.
+* For Istio 1.5.x deployments: update to [Istio 1.5.1](/news/releases/1.4.x/announcing-1.5.1) or later.
```
Comment (fpesce) [inline content/en/news/security/istio-security-2020-004/index.md]:
```suggestion
* For Istio 1.5.x deployments: update to [Istio 1.5.1](/news/releases/1.5.x/announcing-1.5.1) or later.
```

Comment (fpesce) [inline content/en/news/security/istio-security-2020-004/index.md]:
```suggestion
* For Istio 1.5.x deployments: update to [Istio 1.5.1](/news/releases/1.5.x/announcing-1.5.1) or later.
```

**Pull request merged**